fem/linearform.cpp

```diff
@@ -137,7 +137,7 @@ void LinearForm::Assemble()
          }
          Array<int> &bdr_marker = *flfi_marker[k];
          MFEM_ASSERT(bdr_marker.Size() == bdr_attr_marker.Size(),
-                     "invalid boundary marker for boundary face integrator #"
+                     "invalid boundary marker for boundary integrator #"
```

Comment:
The text of the error message was correct: "boundary integrator" is different from "boundary face integrator" and this loop is processing the latter.